tests-beta/android/MASVS-CODE/MASTG-TEST-0x80.md

```diff
@@ -0,0 +1,26 @@
+---
+Title: Testing Enforced Updating
+ID: MASTG-TEST-0x80
+Link: https://mas.owasp.org/MASTG/tests/android/MASVS-CODE/MASTG-TEST-0080/
+Platform: android
+MASVS v1: ['MSTG-ARCH-9']
+MASVS v2: ['MASVS-CODE-2']
+type: [static]
+---
+
+## Overview
+
+When a vulnerability is found in the app, it should be possible to force the user to update the application to continue using it.
+
+## Steps
+
+1. Examine the startup flow of the application. Identify if the application calls out to a backend with the application's version information included.
+2. Examine if the application can handle a specific response from the backend indicating that the application must be updated. For example, the application might evaluate the response from the backend and show a specific error message. Note that the error message can also come from the backend, so the lack of a custom error message in the application does not mean that the feature isn't implemented.
```

Comment:
Perhaps this part should be clarified? Perhaps it could be written like this:

2. Examine if the application can handle a specific response from the backend indicating that the application must be updated. For example by showing or evaluating an error message retrieved from an API or the mobile application.